CONTRIBUTING.md

```diff
@@ -132,4 +132,21 @@ is built. These files are:
 * iOS: `/Realm.xcodeworkspace`: Open in XCode and add the files to RealmJS under `Build Phases`
 * Node: `/realm.gypi`
 
+### How To: Debug React Native Unit tests
 
+This guide assumes that development is happening on a Mac.
+
+When developing new functionality, unit- and integration tests are being run on many different platforms. One of them being React Native.
+If a bug occurs on this platform, it is, unfortunately, rather difficult to debug due to the complex nature of this repository and the React Native build system.
+
+Debugging and working with the unit tests in an iterative mannner is done the following way:
+
+1. Run `./scripts/test.sh react-tests` to install all the dependencies.
+2. Run `./node_modules/.bin/ros start --data realm-object-server-data` in one terminal window.
+3. `cd tests/react-test-app && npm start` in another terminal window
+4. Open `tests/react-test-app/ios/ReactTests.xcworkspace` (note: not the xcodeproj) in Xcode.
+5. Hit Cmd-U to run the tests.
+
+If want to modify the Javascripts in an iterative manner or enable break points you need to do it on the files located in `tests/react-test-app/node_modules/realm-tests`. These files are a copy of the original files located in `tests/js` so any changes must manually be copied back. The reason for this is that the React Native Metro Bundler doesn't support symlinks.
+
+Note that it isn't possible to easily run a single unit test from Xcode. Instead you should disable the tests manually by modifying `tests/react-test-app/node_modules/realm-tests/index.js`.
```

Comment:
Not really directly relevant to this, but it used to be possible to run individual tests from the Xcode test UI and I'm not sure when it stopped working (it now just skips all tests if you try to run a subset).